modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -537,6 +537,12 @@ The DataWeave (weave) format is the canonical format for all transformations.
 This format can help you understand how input data is interpreted before it is
 transformed to a new format.
 
+[WARNING]
+====
+This format is intended only as a help for debugging the results of DataWeave transformations.
+It is significantly slower than other formats. It is not recommended to be used in production applications because it can impact the performance.
```

Comment:
It is significantly slower than other formats and is not recommended for use in production applications because it can impact performance.

Reply:
Ok, sounds good.